pkgs/leak_tracker/lib/src/leak_tracking/_baseliner.dart

```diff
@@ -2,8 +2,6 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// ignore_for_file: avoid_print
```

Comment:
@polina-c - there are two `print` statements in this file; I suspect they're bugs (should be removed? the output routed to a file / to devtools?).

Reply:
They are not bugs. It is functionality, that will be used for tests. Tests are expected to  output the information.

Reply:
To stdout? What if users call that code? We'll produce random lines to their console.

Reply:
This will output will happen if only user asked this to be outputed. Intentionally. The tests will output performance metrics.

Reply:
OK, understood.

FYI that we are no longer reporting `avoid_print` (print uses) in this file in any case, as the linting is not based on flutter/lints.